### PR TITLE
Add default endpoints param

### PR DESCRIPTION
**Pull request by kaiogu**
**Proposed changes**:
- Added enpoints.yaml as default, as it was loaded by default.
- fix https://github.com/RasaHQ/rasa/issues/7445

**READ**
This is most probably not the solution you want for issue #7445 , I opened just to start a discussion.
Even though the default _is_ `None`, the `endpoints.yml` file gets loaded by default in a number of occasions.
In my case, it was in `_get_credentials_and_endpoints_paths()` in `rasa/cli/x.py` which got called by `start_rasa_for_local_rasa_x()` in the same file. There are a number of occurences like this.
This does not seem to be a critical issue, it just might confuse some people. More thoght and effort would be needed to fix this properly.

**Status (please check what you already did)**:
- [ ] added some tests for the functionality
- [ ] updated the documentation
- [ ] updated the changelog (please check [changelog](https://github.com/RasaHQ/rasa/tree/master/changelog) for instructions)
- [ ] reformat files using `black` (please check [Readme](https://github.com/RasaHQ/rasa#code-style) for instructions)

### CODE REVIEW

**Comment by sara-tagger** [comment]
Thanks for submitting a pull request 🚀 @joejuzl will take a look at it as soon as possible ✨

**Comment by joejuzl** [comment]
Hmm yes - the issue with this would be that we no longer get the behaviour of `get_validated_path` which warns the user that the parameter was not set and the default is being used.  Is that warning not enough? Or do you think it needs to be reflected in the help text too?

**Comment by kaiogu** [comment]
If something gets loaded anyway behind the scenes, why not set it as the default, instead of warning and setting it to something not in the docs?

If I remember correctly, I wanted to test rasa without the action server, and it did not behave as expected, as it loaded the file.

**Comment by joejuzl** [comment]
@kaiogu you are right - it should be explicit.

**Comment by wochinge** [comment]
I agree with the fix and think it's a very good point. I also can't think of any scenarios where you'd not want Rasa Open Source to connect to an action server. Any custom action requests would just fail in this case. And as we create the connection per action, we don't create a connection without need in case there is not custom action.

**Comment by joejuzl** [comment]
@kaiogu can you fix the code quality CI check?

**Comment by kaiogu** [comment]
@joejuzl Sure, will do it this weekend.

**Comment by joejuzl** [comment]
Hey @kaiogu, will you have time for this? :) 
Or would you like me to take it over?

**Comment by kaiogu** [comment]
Hey @joejuzl, I'm juggling with too many things at once at the moment, so if you could take over, it would be great!

File: rasa/cli/arguments/default_arguments.py [hunk 1]
```diff
@@ -90,8 +91,9 @@ def add_out_param(
 def add_endpoint_param(
     parser: Union[argparse.ArgumentParser, argparse._ActionsContainer],
     help_text: Text,
-    default: Optional[Text] = None,
+    default: Optional[Text] = DEFAULT_ENDPOINTS_PATH,
 ) -> None:
+    """Add an option to an argument parser to configure endpoints path."""
```
Comment (wochinge) [inline rasa/cli/arguments/default_arguments.py:96]:
```suggestion
    """Adds an option to an argument parser to configure endpoints path."""
```

**Review by wochinge — APPROVED** [review]

**Review by joejuzl — APPROVED** [review]

**Comment by joejuzl** [comment]
> [CLA assistant check](https://cla-assistant.io/RasaHQ/rasa?pullRequest=7523)
> Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you all sign our [Contributor License Agreement](https://cla-assistant.io/RasaHQ/rasa?pullRequest=7523) before we can accept your contribution.
> **1** out of **2** committers have signed the CLA.
> 
> ✅ joejuzl
> ❌ Kaio Utsch
> 
> **Kaio Utsch** seems not to be a GitHub user. You need a GitHub account to be able to sign the CLA. If you have already a GitHub account, please [add the email address used for this commit to your account](https://help.github.com/articles/why-are-my-commits-linked-to-the-wrong-user/#commits-are-not-linked-to-any-user).
> You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/RasaHQ/rasa?pullRequest=7523) it.

Hi @kaiogu - could you sign the CLA? then we can merge this! 😁

**Comment by kaiogu** [comment]
@joejuzl done!

**Comment by HotThoughts** [comment]
Hey @kaiogu, seems like CLAassistant couldn't find your signature under the name Kaio Utsch.

Did you add the email address (same as the one you used for you GitHub account) as suggested?
> **Kaio Utsch** seems not to be a GitHub user. You need a GitHub account to be able to sign the CLA. If you have already a GitHub account, please [add the email address used for this commit to your account](https://help.github.com/articles/why-are-my-commits-linked-to-the-wrong-user/#commits-are-not-linked-to-any-user).
> You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/RasaHQ/rasa?pullRequest=7523) it.

Once you assigned, GitHub will merge your PR.

**Comment by kaiogu** [comment]
> Hey @kaiogu, seems like CLAassistant couldn't find your signature under the name Kaio Utsch.
> 
> Did you add the email address (same as the one you used for you GitHub account) as suggested?
> 
> > **Kaio Utsch** seems not to be a GitHub user. You need a GitHub account to be able to sign the CLA. If you have already a GitHub account, please [add the email address used for this commit to your account](https://help.github.com/articles/why-are-my-commits-linked-to-the-wrong-user/#commits-are-not-linked-to-any-user).
> > You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/RasaHQ/rasa?pullRequest=7523) it.
> 
> Once you assigned, GitHub will merge your PR.

@HotThoughts After following that link I think the following applies:


> Unrecognized author (no email address) If you see this message without an email address, you used a generic email address that can't be connected to your account on GitHub. You will need to set your commit email address in Git, then add the new address to your GitHub email settings to link your future commits. Old commits will not be linked.

As I see this message without an email, it seems I can't get this commit to link to my GitHub account. There seems to be an alternative however (as per the same link you provided):

> If your local Git configuration contained a generic email address, or an email address that was already attached to another user's account, then your previous commits will not be linked to your account. While Git does allow you to change the email address used for previous commits, we strongly discourage this, especially in a shared repository.

**Comment by HotThoughts** [comment]
> > Hey @kaiogu, seems like CLAassistant couldn't find your signature under the name Kaio Utsch.
> > Did you add the email address (same as the one you used for you GitHub account) as suggested?
> > > **Kaio Utsch** seems not to be a GitHub user. You need a GitHub account to be able to sign the CLA. If you have already a GitHub account, please [add the email address used for this commit to your account](https://help.github.com/articles/why-are-my-commits-linked-to-the-wrong-user/#commits-are-not-linked-to-any-user).
> > > You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/RasaHQ/rasa?pullRequest=7523) it.
> > 
> > 
> > Once you assigned, GitHub will merge your PR.
> 
> @HotThoughts After following that link I think the following applies:
> 
> > Unrecognized author (no email address) If you see this message without an email address, you used a generic email address that can't be connected to your account on GitHub. You will need to set your commit email address in Git, then add the new address to your GitHub email settings to link your future commits. Old commits will not be linked.
> 
> As I see this message without an email, it seems I can't get this commit to link to my GitHub account. There seems to be an alternative however (as per the same link you provided):
> 
> > If your local Git configuration contained a generic email address, or an email address that was already attached to another user's account, then your previous commits will not be linked to your account. While Git does allow you to change the email address used for previous commits, we strongly discourage this, especially in a shared repository.

The email address associated with the commit is utsch@buhl-data.com. Is this email address linked with your GitHub account?

**Comment by kaiogu** [comment]
@HotThoughts Added the E-mail

**Pull request merged**